pkg/gofr/context_test.go

```diff
@@ -78,6 +78,8 @@ func TestContext_AddTrace(t *testing.T) {
 }
 
 func TestContext_WriteMessageToSocket(t *testing.T) {
+	t.Skip("skipping for now as the test is flaky and needs to be fixed")
```

Comment:
You can use t.Short and launch the CI test with -short

This way tests will keep running locally